04/Task04.js

```diff
@@ -1,29 +1,79 @@
+import Tab from '../src/components/Tabs/Tab';
+import TabContent from '../src/components/Tabs/TabContent';
+import TabsContent from '../src/components/Tabs/TabsContent';
 
 const Task04 = () => {
-    return (
-        <Row>
-            <Col>
-                <RBTabs defaultActiveKey="profile" id="uncontrolled-tab-example">
-                    <RBTab eventKey="home" title="Home">
-                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur condimentum lacus nec ligula faucibus rhoncus. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; </p>
-                    </RBTab>
-                    <RBTab eventKey="profile" title="Profile">
-                        <p>Donec dignissim ultricies felis, eu dictum eros congue in. In gravida lobortis libero nec tempus. Cras rutrum nisl ut leo volutpat rhoncus. Nulla massa nulla, viverra hendrerit laoreet at, tincidunt eu lacus.</p>
-                    </RBTab>
-                    <RBTab eventKey="contact" title="Contact" disabled>
-                        <p>Vivamus metus nulla, fermentum eget placerat vitae, mollis interdum elit. Pellentesque arcu augue, vulputate ut porttitor ut, suscipit non orci. Integer justo odio, suscipit eget tortor nec, molestie lobortis eros. Nullam commodo elit sit amet lacus blandit aliquet. Mauris at nibh eget nisl pulvinar dignissim.</p>
-                    </RBTab>
-                </RBTabs>
-            </Col>
-            <Col>
-                Tabs!
-            </Col>
-        </Row>
-    )
-}
+  return (
+    <Row>
+      <Col>
+        <RBTabs defaultActiveKey='profile' id='uncontrolled-tab-example'>
+          <RBTab eventKey='home' title='Home'>
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur
+              condimentum lacus nec ligula faucibus rhoncus. Vestibulum ante
+              ipsum primis in faucibus orci luctus et ultrices posuere cubilia
+              Curae;
+            </p>
+          </RBTab>
+          <RBTab eventKey='profile' title='Profile'>
+            <p>
+              Donec dignissim ultricies felis, eu dictum eros congue in. In
+              gravida lobortis libero nec tempus. Cras rutrum nisl ut leo
+              volutpat rhoncus. Nulla massa nulla, viverra hendrerit laoreet at,
+              tincidunt eu lacus.
+            </p>
+          </RBTab>
+          <RBTab eventKey='contact' title='Contact' disabled>
+            <p>
+              Vivamus metus nulla, fermentum eget placerat vitae, mollis
+              interdum elit. Pellentesque arcu augue, vulputate ut porttitor ut,
+              suscipit non orci. Integer justo odio, suscipit eget tortor nec,
+              molestie lobortis eros. Nullam commodo elit sit amet lacus blandit
+              aliquet. Mauris at nibh eget nisl pulvinar dignissim.
+            </p>
+          </RBTab>
+        </RBTabs>
+      </Col>
+      <Col>
+        <Tabs>
```

Comment:
Zwróć uwagę, że Twój sposób zapisu tego komponentu powoduje, że trzeba powielać `title` to może powodować, że ktoś się "machnie", źle wpisze i już nie będzie działać. Opcja powyżej tego nie ma i taka wersja jest lepsza ;)